client/webserver/site/src/css/utilities.scss

```diff
@@ -0,0 +1,189 @@
+.flex-column {
+  flex-direction: column;
+}
```

Comment:
I may have added this while hacking at some point, but I believe this is already defined in bootstrap.scss

Reply:
Yeah though we do need this now that only a subset of the utilities are imported.